src/utils/domUtils.ts

```diff
@@ -20,13 +20,13 @@ export module Utils {
       return bbox;
     }
 
-    export var POLYFILL_TIMEOUT_MILLISECONDS = 1000 / 60; // 60 fps
+    export var REQUEST_ANIMATION_FRAME_TIMEOUT_MILLISECONDS = 1000 / 60; // 60 fps
 
     export function requestAnimationFramePolyfill(fn: () => void) {
```

Comment:
Based on the first google result of "polyfill" you get...

"A polyfill, or polyfiller, is a piece of code (or plugin) that provides the technology that you, the developer, expect the browser to provide natively." - https://remysharp.com/2010/10/08/what-is-a-polyfill

Which basically says that the word polyfill here means almost nothing.  Our `Map`s by this definition are I think polyfills and thus having the variable name `POLYFILL_TIMEOUT_MILLISECONDS` I think is really not useful